js/main.js

```diff
@@ -26,7 +26,8 @@ jQuery(function($) {
 				var $this = $(this);
 
 				// add a tab head link for each tab
-				tabsHead.append('<a href="#">' + $this.find('.gsection_title').html() + '</a>');
+				// add the id as well so gf conditional sections still works
+				tabsHead.append('<a id="' + $this[0].id + '" href="#">' + $this.find('.gsection_title').html() + '</a>');
```

Comment:
Hey guys -- I'm new to GitHub, so apologize if this is a stupid question. It appears you've made some modifications that fixes those of us that have "conditional sections".  How do I get the modified code?   Is the modified code limited to just js/main.js and it's just a matter of copying the file?  What is a pull request?  At what point does a new "version" of the plugin become available with the modified code?